packages/account-v2/.stylelintrc.json

```diff
@@ -0,0 +1 @@
+{ "extends": ["stylelint-config-standard"] }
```

Comment:
stylelint-config-standard-scss if we are going to use scss

Reply:
I am unable to stylelint-config-standard-scss due to the hoisted version of stylelint version used in deriv app

Reply:
tailwind + scss is a bit overhead IMO, we should try to use tailwind classes or if needed we can opt-in for css modules directly rather than adding scss compiler on top of tailwind 
What do you think

Reply:
if we are going to use tailwind, make sense. as I understand they don’t recommend using preprocessors